docs/_posts/2022-12-21-jekyll-sass-converter-3.0-released.markdown

```diff
@@ -0,0 +1,71 @@
+---
+title: Jekyll Sass Converter 3.0 Released
+date: 2022-12-21 17:52:15 +0530
+author: ashmaroli
+category: community
+---
+
+Jekyll Sass Converter 3.0 shipped recently, is available to those using Jekyll 4.3 and above and contains major changes.
+Specifically, the plugin has **stopped using `sassc` for converting your Sass partials and stylesheets** into CSS files.
+Instead, the converter now uses `sass-embedded` gem acting as an interface to Dart Sass, which is the current primary
+implementation of Sass under active development. The secondary implementation `libsass` which `sassc` gem interfaced
+with has been deprecated by its developers.
+
+However, Dart Sass isn't *fully compatible* with older Ruby Sass workflow.
+
+## Requirements
+
+- Minimum Ruby Version: `Ruby 2.6.0` (all platforms).
+- Minimum Rubygems Version: `3.3.22` (for Linux-based platforms).
+
+## Migration Guide
+
+### Dropped `implmentation` Option
+
+In `v3.0.x`, `sass-embedded` is the only supported Sass implmentation, and therefore the config option
+`sass.implementation` introduced in `v2.2.0` has been removed.
+
+
+### Dropped `add_charset` Option
+
+The converter will no longer emit `@charset "UTF-8";` or a `U+FEFF` (byte-order marker) for `sassify` and `scssify`
+Jekyll filters and hence the redundant option `sass.add_charset` is no longer active.
+
+
+### Dropped `line_comments` Option
+
+`sass-embedded` does not support `sass.line_comments` option.
+
+
+### Dropped support of importing files with non-standard extension names
+
+`sass-embedded` only allows importing files that have extension names of `.sass`, `.scss` or `.css`. **SCSS syntax in
+files with `.css` extension name will result in a syntax error**.
```

Comment:
The bold here doesn't have the same impact since it's surrounded by all the headings and I think we can simply take it out.

```suggestion
`sass-embedded` only allows importing files that have extension names of `.sass`, `.scss` or `.css`. SCSS syntax in
files with `.css` extension name will result in a syntax error.
```